### PR TITLE
Add graph service functions tests

### PR DESCRIPTION
**Pull request by Plastova**

### CODE REVIEW

**Review by orrrrtem — DISMISSED** [review]
static asserts related comments need to be removed.
Incorrect results to be reported at least in the backlog.

File: cpp/oneapi/dal/graph/test/badarg.cpp [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+        std::allocator<char>>(graph_data.get_vertex_count(),
+                              graph_data.get_edge_count(),
+                              graph_data.rows.data(),
+                              graph_data.cols.data(),
+                              graph_data.edge_weights_int.data());
+    const auto &g = graph_builder.get_graph();
+    REQUIRE_THROWS_AS(dal::preview::get_edge_value(g, 1, 7), out_of_range);
+}
+
+//incorrect result
```
Comment (bysheva) [inline cpp/oneapi/dal/graph/test/badarg.cpp:195]: Probably need to remove these comments

File: cpp/oneapi/dal/graph/test/badarg.cpp [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+        std::allocator<char>>(graph_data.get_vertex_count(),
+                              graph_data.get_edge_count(),
+                              graph_data.rows.data(),
+                              graph_data.cols.data(),
+                              graph_data.edge_weights_int.data());
+    const auto &g = graph_builder.get_graph();
+    REQUIRE_THROWS_AS(dal::preview::get_edge_value(g, 0, -10), out_of_range);
+}
+
+// SERVICE_FUNCTIONS_BADARG_TEST("Directed graph for get_vertex_degree") {
```
Comment (bysheva) [inline cpp/oneapi/dal/graph/test/badarg.cpp:260]: Also to remove unnecessary comments

**Review by bysheva — CHANGES_REQUESTED** [review]

**Review by bysheva — APPROVED** [review]

**Pull request merged**